types/robust-point-in-polygon/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for robust-point-in-polygon 1.0
+// Project: https://github.com/mikolalysenko/robust-point-in-polygon
+// Definitions by: Crash Springfield <https://github.com/crashspringfield>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type Point = [number, number];
+
+declare function robustPointInPolygon(vs: Point[], point: Point): number;
+
+export {
+  Point,
+  robustPointInPolygon
```

Comment:
This is not the same thing as what the package does, and won't work at runtime. You really want something closer to

```ts
type Point = [number, number];

declare function robustPointInPolygon(vs: Point[], point: Point): number;
export = robustPointInPolygon;
```

or

```ts
declare namespace robustPointInPolygon {
    type Point = [number, number];
}

declare function robustPointInPolygon(vs: Point[], point: Point): number;
export = robustPointInPolygon;
```

if exporting the `Point` type is that useful.

Then you can use `import robustPointInPolygon = require("robust-point-in-polygon");`

Reply:
I've gone ahead and updated it with your suggestions, and instead of importing/exporting the Point type, just re-declared it in the test so the test would run. Thanks for the help!